docs/api/openapi.yaml

```diff
@@ -0,0 +1,217 @@
+openapi: 3.0.3
+
+info:
+  contact:
+    email: harvester@dans.knaw.nl
+  description: >-
+    Voor het eenvoudig ophalen van een publicatie of van data met een in Nederland toegekende NBN is er de <a href
```

Comment:
some of the description is in dutch, some in english

Reply:
Wow!